android/src/main/java/com/pdftron/reactnative/views/DocumentView.java

```diff
@@ -769,10 +769,6 @@ public void setAnnotationToolbars(ReadableArray toolbars) {
                 if (isValidToolbarTag(tag)) {
                     AnnotationToolbarBuilder toolbarBuilder = DefaultToolbars
                             .getDefaultAnnotationToolbarBuilderByTag(tag);
-                    // SDK Support Issue 22893
-                    // To ensure if the client changes the order of the annotation tools that the UI
-                    // will reflect the changed state
-                    mBuilder = mBuilder.addToolbarBuilder(toolbarBuilder).saveToolbarItemOrder(false);
```

Comment:
Need to keep addToolbarBuilder

Reply:
Restored mBuilder part as per original PR:
https://github.com/PDFTron/pdftron-react-native/pull/292/files